de.tudarmstadt.ukp.wikipedia.mwdumper/src/main/java/de/tudarmstadt/ukp/wikipedia/mwdumper/importer/XmlDumpReader.java

```diff
@@ -33,6 +33,7 @@
 import java.util.Map;
 import java.util.TimeZone;
 
+import javax.xml.XMLConstants;
```

Comment:
Right, I can confirm. Interesting. Since this is unrelated to the actual PR, would you mind of I fix this on the master directly, and you exclude it from the PR?